client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -281,28 +281,28 @@ function mediaButton( editor ) {
 		renderDropZone( { visible: event.type === 'dragend' } );
 	}
 
+	editor.addCommand( 'wpcomAddMedia', () => {
+		var selectedSite = sites.getSelectedSite();
```

Comment:
@gwwar just because I was trying not to change the code

you think I should change it?